client/src/App.tsx

```diff
@@ -10,6 +11,8 @@ function App(): JSX.Element {
       <Switch>
         <Route path='/donation-guidelines'><SampleContainer /></Route>
         <Route path='/login'><SampleContainer /></Route>
+        {/* TODO: delete /confirmation endpoint after logic for triggering the confirmation modal upon signup has been added */}
```

Comment:
could you leave your username next to the TODO comment (something like `TODO(jlight99):...`) so we know who to talk to if we have questions